Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'yard-activesupport-concern'
 gem 'origen_jtag', '>= 0.12.0'
 gem 'origen_doc_helpers'
 gem 'nokogiri', '1.8.5'  # Lock to the version to enable testing in Ruby 2.2
+gem 'dry-inflector', '0.1.2'
```

Comment:
Can you add a comment as to why this is here so we don't forget

Reply:
added